snippets/freeipa_register.erb

```diff
@@ -24,6 +24,9 @@ name: freeipa_register
 #
 #   freeipa_opts                Additional options to pass directly to installer
 #
+
+/usr/sbin/sshd-keygen
```

Comment:
For interest, I unpacked the openssh-server package - digging into the package, the postinst calls `ssh-keygen -q -f "$file" -N '' "$@"`, iterating over the various keys types. Basically the same, you just get the convenience of it doing all the key types in one command.